spec/requests/api/requests_spec.rb

```diff
@@ -228,6 +228,24 @@
       expect(response.parsed_body).to include(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes workflow in the request resources" do
+      ems = FactoryGirl.create(:ems_vmware)
+      vm_template = FactoryGirl.create(:template_vmware, :name => "template1", :ext_management_system => ems)
+      request = FactoryGirl.create(:miq_provision_request,
+                                   :requester => @user,
+                                   :src_vm_id => vm_template.id,
+                                   :options   => {:owner_email => 'tester@example.com'})
+
+      MiqDialog.seed
```

Comment:
I saw some other tests had `MiqDialog.seed` and decided to replicate that. But you're right, it would be an overkill.
I've addressed this in the last commit. Thanks.